docs/user-interface/controls/blazorwebview.md

```diff
@@ -0,0 +1,125 @@
+             xmlns:x=http://schemas.microsoft.com/winfx/2009/xaml
+             xmlns:b="clr-namespace:Microsoft.AspNetCore.Components.WebView.Maui;assembly=Microsoft.AspNetCore.Components.WebView.Maui"
+             xmlns:local="clr-namespace:BlazorWebViewDemo"
+             x:Class="BlazorWebViewDemo.MainPage"
+             BackgroundColor="{DynamicResource PageBackgroundColor}">
+
+    <b:BlazorWebView HostPage="wwwroot/index.html">
+        <b:BlazorWebView.RootComponents>
+            <b:RootComponent Selector="app" ComponentType="{x:Type local:Main}" />
+        </b:BlazorWebView.RootComponents>
+    </b:BlazorWebView>
+
+</ContentPage>
+```
+
+The root Blazor component for the app is in *Main.razor*, which Razor compiles into a type named `Main` in the application's root namespace. The rest of the Blazor components are in the *Pages* and *Shared* project folders, and are identical to the components used in the default Blazor web template. Static web assets for the app are in the *wwwroot* folder.
+
+## Add a BlazorWebView to an existing app
+
+The process to add a `BlazorWebView` to an existing .NET MAUI app is as follows:
+
+1. Add the Razor SDK, `Microsoft.NET.Sdk.Razor` to your project by editing its first line of the CSPROJ project file:
+
+    ```xml
+    <Project Sdk="Microsoft.NET.Sdk.Razor">
+    ```
+
+    The Razor SDK is required to build and package projects containing Razor files for Blazor projects.
+
+1. Add the root Blazor component for the app to the project.
+1. Add your Blazor components to project folders named *Pages* and *Shared*.
+1. Add your static web assets to a project folder named *wwwroot*.
+1. Add any optional *_Imports.razor* files to your project.
+1. Add a `BlazorWebView` to a page in your .NET MAUI app, and point it to the root of the Blazor app:
+
+    ```xaml
+    <ContentPage xmlns=http://schemas.microsoft.com/dotnet/2021/maui
+                 xmlns:x=http://schemas.microsoft.com/winfx/2009/xaml
+                 xmlns:b="clr-namespace:Microsoft.AspNetCore.Components.WebView.Maui;assembly=Microsoft.AspNetCore.Components.WebView.Maui"
+                 xmlns:local="clr-namespace:MyBlazorApp"
+                 x:Class="MyBlazorApp.MainPage">
+
+        <b:BlazorWebView HostPage="wwwroot/index.html">
+            <b:BlazorWebView.RootComponents>
+                <b:RootComponent Selector="app" ComponentType="{x:Type local:Main}" />
+            </b:BlazorWebView.RootComponents>
+        </b:BlazorWebView>
+
+    </ContentPage>
+    ```
+
+1. Modify your `MauiProgram` class to add `using` directives for the `Microsoft.AspNetCore.Components.WebView.Maui` and `Microsoft.Extensions.DependencyInjection` namespaces:
+
+    ```csharp
+    using Microsoft.AspNetCore.Components.WebView.Maui;
+    using Microsoft.Extensions.DependencyInjection;
+    ```
+
+    > [!NOTE]
+    > Blazor requires service scopes, which are only supported with the `Microsoft.Extensions.DependencyInjection` library.
```

Comment:
I think this note is no longer needed; the default MAUI stuff has everything you need.